docs/src/gauss_newton_kalman_inversion.md

```diff
@@ -0,0 +1,57 @@
+# [Gauss Newton Kalman Inversion](@id gnki)
+
+### What Is It and What Does It Do?
+Gauss Netwon Kalman Inversion (GNKI) ([Chada et al, 2020](https://arxiv.org/pdf/2010.13299)), also known as the Iterative Ensemble Kalman Filter with Statistical Linearization, is a derivative-free ensemble optimizaton method based on the Gauss Newton optimization update and the Iterative Extended Kalman Filter (IExKF) ([Jazwinski, 2007]).  In the linear case and continuous limit, GNKI recovers the true posterior mean and covariance.  Empirically, GNKI performs well as an optimization algorithm in the nonlinear case.  
+
+### Problem Formulation
+
+The data ``y`` and parameter vector ``\theta`` are assumed to be related according to:
+```math
+\tag{1} y = \mathcal{G}(\theta) + \eta \,,
+```
+where ``\mathcal{G}:  \mathbb{R}^p \rightarrow \mathbb{R}^d`` denotes the forward map, ``y \in \mathbb{R}^d`` is the vector of observations, and ``\eta`` is the observational noise, which is assumed to be drawn from a ``d``-dimensional Gaussian with distribution ``\mathcal{N}(0, \Gamma_y)``. The objective of the inverse problem is to compute the unknown parameters ``\theta`` given the observations ``y``, the known forward map ``\mathcal{G}``, and noise characteristics ``\eta`` of the process.
+
+!!! note
+    GNKI relies on minimizing a loss function that includes regularization.  The user must specify a Gaussian prior distribution. See [Prior distributions](@ref parameter-distributions) to see how one can apply flexible constraints while maintaining Gaussian priors. 
```

Comment:
maybe introduce the m_\theta, \Gamma_\theta notation here?

Reply:
I tried introduced it in the note (and kept it in the same place).  Do you think changing $m$ to $m_{\theta}$ would be easier to understand?  I thought it might look clunky with the \theta at the end and also having the ensemble member and iteration super and sub scripts.